equinix/resource_fabric_cloud_router.go

```diff
@@ -15,47 +18,250 @@ import (
+		},
+		"package": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Fabric Cloud Router Package Type",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: FabricCloudRouterPackageSch(),
+			},
+		},
+		"change_log": {
+			Type:        schema.TypeSet,
+			Computed:    true,
+			Description: "Captures Fabric Cloud Router lifecycle change information",
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.ChangeLogSch(),
+			},
+		},
+		"type": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validation.StringInSlice([]string{"XF_ROUTER"}, true),
+			Description:  "Defines the FCR type like XF_ROUTER",
+		},
+		"location": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Fabric Cloud Router location",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.LocationSch(),
+			},
+		},
+		"project": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Customer resource hierarchy project information.\nApplicable to customers onboarded to Equinix Identity and Access Management. For more information see Identity and Access Management: Projects",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: FabricCloudRouterProjectSch(),
+			},
+		},
+		"account": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Customer account information that is associated with this Fabric Cloud Router",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: FabricCloudRouterAccountSch(),
+			},
+		},
+		"order": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: "Order information related to this Fabric Cloud Router",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.OrderSch(),
+			},
+		},
+		"notifications": {
+			Type:        schema.TypeList,
+			Required:    true,
+			Description: "Preferences for notifications on Fabric Cloud Router configuration or status changes",
+			Elem: &schema.Resource{
+				Schema: equinix_fabric_schema.NotificationSch(),
+			},
+		},
+		"bgp_ipv4_routes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Access point used and maximum number of IPv4 BGP routes",
+		},
+		"bgp_ipv6_routes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Access point used and maximum number of IPv6 BGP routes",
+		},
+		"distinct_ipv4_prefixes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Number of IPv4 BGP routes in use (including non-distinct prefixes).",
+		},
+		"distinct_ipv6_prefixes_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Number of IPv6 BGP routes in use (including non-distinct prefixes)",
+		},
+		"connections_count": {
+			Type:        schema.TypeInt,
+			Computed:    true,
+			Description: "Number of connections associated with this Access point",
```

Comment:
There isn't any access point referred to here. We should label it as just a cloud router, but double check with documentation.

Reply:
It's Total number of connections to the given Fabric Cloud Router instance. I have updated it in the schema description.